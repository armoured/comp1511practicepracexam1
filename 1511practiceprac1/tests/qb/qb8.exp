1 3 8 9 20 22 43 52 76 100 
