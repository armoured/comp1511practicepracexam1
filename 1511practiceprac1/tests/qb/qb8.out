20 3 100 76 8 22 43 52 9 1 
